lapce-data/src/command.rs

```diff
@@ -759,6 +763,7 @@ pub enum LapceUICommand {
         apply_naming: bool,
     },
     FileExplorerRefresh,
+    CopyString(String),
```

Comment:
Is it better to rename is something like ```PutToClipyboard```

Reply:
is it worth changing LapceUICommand::CopyPath and LapceUICommand::CopyRelacivePath to use PutToClipboard?

Reply:
I think that would be good